doh/_parseURLargs.js

```diff
@@ -190,20 +182,30 @@
 				'dojox'
 			],
 			deps: ["dojo/domReady", "doh/main"],
-			callback: function(domReady, doh){
-				domReady(function(){
-					doh._fixHeight();
-					doh.breakOnError= breakOnError;
-					require(test, function(){
-						doh.run();
-					});
-				});
-			},
+			callback: callback,
 			async: async,
 			isDebug: 1
 		};
 	}
 	
+	function callback(domReady, doh){
+		domReady(function(){
+			doh._fixHeight();
+			doh.breakOnError= breakOnError;
+			for (var amdTests = [], i = 0, l = test.length; i < l; i++) {
```

Comment:
`amdTests` should be declared at the top of the function scope, as it is used later, outside of this block.